config/example_cnn_daily_batch_size.yaml

```diff
@@ -0,0 +1,31 @@
+name: "CNN Daily main"
+training_data:
+  dataset: "abisee/cnn_dailymail"
+  input_feature: "article"
+  label_feature: "highlights"
+  split_name: "train"
+  textual_labels: []
+  dataset_version: "1.0.0"
```

Comment:
@gkaretka what's this about?

Reply:
just specifying dataset version for `datasets` library (`load_dataset`)